pkg/rpc/server/server.go

```diff
@@ -800,7 +793,7 @@ func getTimestampsAndLimit(ps request.Params, index int) (uint64, uint64, int, i
 		end = uint64(time.Now().Unix() * 1000)
 	}
 	if pStart != nil {
-		val, err := pStart.GetInt()
+		val, err := pStart.AsInt()
```

Comment:
Fixed, now these methods keep the old names. As a result, the convention is the following:
1. GetInt() / GetString() / GetBoolean() are non-strict, i.e. they try to unmarshal raw message into all possible types and then cast this type to desired one.
2. GetIntStrict() / GetStringStrict() / GetBooleanStrict() are strict, i.e. they try to unmarshal raw message into corresponding type and fail on error.

Most of use-cases are covered by non-strict wrappers, but I still would like to keep strict wrappers, because there are several methods where we need strict conversions and unmarshalling-related code is not something I'd like to see there. These methods are `CreateFunctionInvocationScript`, `contractIDFromParam`, `blockHashFromParam`, `getStateRoot`.